content/system76-software.md

```diff
@@ -157,6 +157,15 @@ paru -S system76-acpi-oled
 
 Be sure to install the <u>System76 Driver</u> first. The steps to do that are [here](/articles/system76-driver).
 
+This command will install all of the packages using <u>Paru</u>.
+
+```bash
+sudo dnf install system76* firmware-manager
```

Comment:
Doesn't this do exactly what lines 169-180 do? (section "System76 Firmware Manager in Fedora")